src/main/java/taxi/dao/jdbc/CarJdbcDao.java

```diff
@@ -0,0 +1,200 @@
+package taxi.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import taxi.dao.CarDao;
+import taxi.lib.Dao;
+import taxi.model.Car;
+import taxi.model.Driver;
+import taxi.model.Manufacturer;
+import taxi.services.util.ConnectionUtil;
+
+@Dao
+public class CarJdbcDao implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (model, manufacturer_id) VALUES (?,?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query,
+                        PreparedStatement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, car.getModel());
+            preparedStatement.setLong(2, car.getManufacturer().getId());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't create car" + car.getModel(), e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String getCarQuery = "SELECT c.id, c.model, m.id, m.name, m.country "
+                + "FROM cars c INNER JOIN manufacturers m ON c.manufacturer_id = m.id"
+                + "WHERE id = ? AND c.deleted = false AND m.deleted = false";
```

Comment:
```suggestion
                + "WHERE c.id = ? AND c.deleted = false AND m.deleted = false";
```
you should be able to solve the mistakes if there are any, and not to avoid them